content/en/docs/tasks/tools/included/optional-kubectl-configs-pwsh.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "powershell auto-completion"
+description: "Some optional configuration for powershell auto-completion."
+headless: true
+---
+
+The kubectl completion script for powershell can be generated with the command `kubectl completion powershell`.
+
+To do so in all your shell sessions, add the following to your `$PROFILE` file:
+
+```powershell
+kubectl completion powershell | Out-String | Invoke-Expression
+```
+
+If you don't want to run kubectl every time, you can just add the generated script to your `$PROFILE` file:
+
+```powershell
+kubectl completion powershell >> $PROFILE
```

Comment:
Can you add a sentence explaining the difference between the two choices?

Reply:
I added a little bit more explanation.

Reply:
great!